### PR TITLE
Update Get-RsRestCacheRefreshPlanHistory.Tests.ps1

### PR DESCRIPTION
**Pull request by SQLvariant**
Changes proposed in this pull request:
 - Increased Start-Sleep duration from 3 seconds to 6 seconds, to allow time for subscription to run and have history to display.

How to test this code:
 - Run the test.

Has been tested on (remove any that don't apply):
 - PowerShell 7.1
 - Windows 10
 - PBIRS

### CODE REVIEW

File: Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1 [hunk 1]
```diff
@@ -61,13 +61,20 @@ Describe "Get-RsRestCacheRefreshPlanHistory" {
             $plan | Should Not BeNullOrEmpty
             $plan.LastStatus | Should Not Be 'New Scheduled Refresh Plan'
 
-            Start-Sleep -Seconds 3
+            Start-Sleep -Seconds 6
```
Comment (jtarquino) [inline Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1:64]: I would remove this wait now that you have the polling

File: Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1 [hunk 2]
```diff
@@ -61,13 +61,20 @@ Describe "Get-RsRestCacheRefreshPlanHistory" {
...
             $someHistory = Get-RsRestCacheRefreshPlanHistory -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
             @($someHistory).count | Should be 1
             
             Start-RsRestCacheRefreshPlan -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
-            Start-Sleep -Seconds 3
-            $moreHistory = Get-RsRestCacheRefreshPlanHistory -ReportPortalUri $reportPortalUri -RsReport "$($rsFolderPath)/ReportCatalog"
+            $timer= get-date
+            while ($true) {
```
Comment (jtarquino) [inline Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1:70]: usually is better practice to have the break conditions in the loop, you could do a Do - while loop instead and add the conditions to the end

Comment (SQLvariant) [reply Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1]: I had tried that earlier but it didn't work.  Turns out adding an extra set of parens fixed it.

File: Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1 (hunk 1 above)
Comment (SQLvariant) [reply Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1]: Done.

**Pull request merged**